src/index.js

```diff
@@ -10,22 +10,18 @@ class Todo {
 }
 class UI {
   static displayTodos = (todos) => {
-    const ul = document.querySelector('ul');
-    if (!todos.length) ul.innerHTML = '';
-    let content = '';
+    const ul = document.querySelector('.list');
+    ul.innerHTML = '';
     todos.forEach((tod) => {
-      content += `
+      ul.innerHTML += `
         <li id=${tod.id} class="item">
           <div class='group'>
-            <input class='bdan' name='checkbox' type="checkbox" />
-            <p contenteditable="true">${tod.desc}</p>
+            <input id=${tod.id} class='bdan' name='checkbox' type="checkbox" />
```

Comment:
Your implementation looks great. Kindly add a condition that checks the checkbox when the task is completed so that the data in storage can be represented correctly on the load of the page